designer/client/src/components/modals/ValidationLabels.tsx

```diff
@@ -22,22 +25,33 @@ const LabelsContainer = styled.div({
 
 export default function ValidationLabels(props: Props) {
     const { validators, values, validationLabelInfo } = props;
+    const { t } = useTranslation();
 
     const validationErrors: ValidationError[] = withoutDuplications(validators)
         .filter((v) => !v.isValid(...values))
         .map((validator) => ({
             message: validator.message && validator.message(),
             description: validator.description && validator.description(),
+            templateValues: validator.templateValues && validator.templateValues(),
+            errorCode: validator.errorCode && validator.errorCode(),
         }));
 
     const isValid: boolean = isEmpty(validationErrors);
 
     const renderErrorLabels = () =>
         validationErrors.map((validationError) => {
+            const { errorCode, templateValues, message } = validationError;
+
+            const canRenderI18nTemplate = errorCode != null && templateValues != null;
+
             // we don't pass description as tooltip message until we make changes on the backend
             return (
-                <LimitedValidationLabel key={validationError.message} title={validationError.message} type="ERROR">
-                    {validationError.message}
+                <LimitedValidationLabel key={message} type="ERROR" title={message}>
```

Comment:
1. The function of this component is to display a tooltip, right? Can we rename it to the <ValidationTooltip />?
2. Do we have a generic way to display tooltips in the application? If not, maybe we should make this component generic and even consider using https://mui.com/material-ui/react-tooltip/ for tooltips. (but for now, making it a shareable component, will be enough IMO)